subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_spi_stm32.c

```diff
@@ -0,0 +1,715 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
This needs a comment documenting/justifying why this driver duplicates the existing STM32 SPI driver.

Reply:
Done.